app/src/main/java/com/example/boylc/demomvp/data/model/Category.java

```diff
@@ -0,0 +1,74 @@
+package com.example.boylc.demomvp.data.model;
+
+import com.example.boylc.demomvp.untils.Constants;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class Category {
+    private int mId;
+    private String mCategoryName;
+    private String mCategoryImage;
+    private String mCategoryImageThump;
+    private int mTotalWallpaper;
+
+    public Category(int mId, String mCategoryName, String mCategoryImage, String mCategoryImageThump, int mTotalWallpaper) {
```

Comment:
https://viblo.asia/p/mot-so-tips-tricks-khi-lam-viec-voi-android-studio-yMnKMJjgZ7P
tip trick số 6
sau khi em làm xong thì xóa constructor, getter, setter đi gen lại cho đúng 
nó sẽ auto chuyển thành 
```suggestion
    public Category(int mId, String categoryName, String categoryImage, String categoryImageThump, int totalWallpaper) {
```